frontend/context/query.tsx

```diff
@@ -25,6 +25,7 @@ type InitialStateType = {
   lastEditedQueryBody: string;
   lastEditedQueryObserverCanRun: boolean;
   lastEditedQueryFrequency: number;
+  lastEditedQueryAutomation: boolean;
```

Comment:
this name implies to me that the value would be a query automation – would
`lastEditedEnableQueryAutomations` be accurate?

Reply:
@jacobshandling would it be better as `lastEditedQueryAutomationsEnabled`? so it matches the query key `automations_enabled`?

Reply:
sounds good